Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -49,17 +49,17 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
             // Don't record uri in 404 errors, to avoid spamming of metrics
             if let endpointPath = context.endpointPath {
                 dimensions = [
-                    ("hb_uri", endpointPath),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", endpointPath),
+                    ("method", request.method.rawValue),
                 ]
-                Counter(label: "hb_requests", dimensions: dimensions).increment()
+                Counter(label: "hb.requests", dimensions: dimensions).increment()
             } else {
                 dimensions = [
-                    ("hb_uri", "NotFound"),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", "NotFound"),
+                    ("method", request.method.rawValue),
```

Comment:
```suggestion
                    ("http.request.route", "NotFound"),
                    ("http.request.method", request.method.rawValue),
```

Reply:
Also the static "NotFound" might not make sense in spec

Reply:
The matched route metadata is `http.route` not `http.request.route`. I changed this to not include the route metadata, but included the error.type to catch 404s